public/js/boards.js

```diff
@@ -1,25 +1,36 @@
-import { getBoards, saveBoard, updateBoard } from "./database.js"
+import { getBoards, saveBoard, updateBoard } from "./database.js";
 
 const trashIcon = "../res/trash-can.svg";
 const pencilIcon = "../res/NotePencil.svg";
 
-
 const db = firebase.database();
 let userId = '';
 let boards = {};
 let boardId = '';
 
+
+
 document.addEventListener("DOMContentLoaded", function() {
     userId = localStorage['userId'] || '';
+
     getBoards(userId, async board => {
         if (boards[board.id] === undefined) { // Display board only once
             boards[board.id] = board;
             await displayBoard(board);
         }
     });
+    setTimeout(function(){ removeElementsByClass('board-skeleton') }, 300);
```

Comment:
Uhm you should display it while your data is being loaded. Not for some arbitrary period.